orix/quaternion/quaternion.py

```diff
@@ -616,6 +687,210 @@ def identity(cls, shape: Union[int, tuple] = (1,)) -> Quaternion:
+            quaternions.
+
+        Examples
+        --------
+        >>> from orix.quaternion import Quaternion
+        >>> q1 = Quaternion([[1, 0, 0, 0], [2, 0, 0, 0]])
+        >>> np.allclose(q1.to_matrix(), np.eye(3))
+        True
+        >>> q2 = Quaternion([[0, 1, 0, 0], [0, 2, 0, 0]])
+        >>> np.allclose(q2.to_matrix(), np.diag([1, -1, -1]))
+        True
+        """
+        q = self.unit
+        a, b, c, d = q.a, q.b, q.c, q.d
+        om = np.zeros(self.shape + (3, 3))
+
+        bb = b**2
+        cc = c**2
+        dd = d**2
+        qq = a**2 - (bb + cc + dd)
+        bc = b * c
+        ad = a * d
+        bd = b * d
+        ac = a * c
+        cd = c * d
+        ab = a * b
+        om[..., 0, 0] = qq + 2 * bb
+        om[..., 0, 1] = 2 * (bc - ad)
+        om[..., 0, 2] = 2 * (bd + ac)
+        om[..., 1, 0] = 2 * (bc + ad)
+        om[..., 1, 1] = qq + 2 * cc
+        om[..., 1, 2] = 2 * (cd - ab)
+        om[..., 2, 0] = 2 * (bd - ac)
+        om[..., 2, 1] = 2 * (cd + ab)
+        om[..., 2, 2] = qq + 2 * dd
+
+        return om
+
+    def to_axes_angles(self) -> Tuple[Vector3d, np.ndarray]:
+        """Return an axis-angle representation of the normalized
+        quaternions.
+        :cite:`rowenhorst2015consistent`.
+
+        Returns
+        -------
+        axis
+            an orix.vector.Vector3D object containing the axes of rotation
+        angle
+            a numpy array representing the angles of rotation
+
+        Examples
+        --------
+        #TODO
+        """
+        return self.axis, self.angle
+
+    def to_rodrigues(self) -> Vector3d:
+        """Return the neo-Eulerian Rodrigues Vector representation of the
+        normalized quaternions.
+        :cite:`rowenhorst2015consistent`.
+
+        Returns
+        -------
+        rod
+            an orix.vector.Vector3D object containing the axes of rotation,
+            with lengths equal to tan(angle/2).
+        Examples
+        --------
+        #TODO
+        Notes
+        --------
+        This is often used as a plotting tool, as it produces an isomorphic
+        (though not volume-preserving) mapping from the non-euclidean
+        orientation space into cartesian coordinates.Additionaly, crystal
+        systems which contain 222 as a quotient group have rectilinear
+        fundamental zones in this representation.
+        """
+        ax = self.axis.unit
+        rod = ax * np.tan(self.angle / 2)
+        return rod
+
+    def to_homochoric(self) -> Vector3d:
+        """Return the neo-Eulerian Homochoric Vector representation of
+        the normalized quaternions.
+        :cite:`rowenhorst2015consistent`.
+
+        Returns
+        -------
+        homo
+            an orix.vector.Vector3D object containing the axes of rotation,
+            with lengths equal to [0.75*(angle-sin(angle))]^(1/3).
+
+        Examples
+        --------
+        #TODO
+        Notes
+        --------
```

Comment:
Sphinx requires exact number of underlines and letters

```suggestion
        -----
```